bash/gast_to_code/bash_gast_to_code_converter.py

```diff
@@ -28,7 +28,7 @@ def handle_arr(gast):
         if general_helpers.arr_in_list(gast["elements"]):
             return "impossibleTranslationError: direct translation does not exist"  # TODO: streamline error message as part of refactor
 
-        return "(" + router.gast_to_code(gast["elements"], "py") + ")"
+        return "(" + router.gast_to_code(gast["elements"], "bash") + ")"
```

Comment:
why is this in this PR?

Reply:
this was causing test failures - think its a bug from another PR